### PR TITLE
trusted roots article split conceptual + how-to

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Court72** [comment]
@robinharwood, @Xelu86

Can you review the proposed changes? 

New content should be submitted via the private repo to benefit from staging, validation, and Acrolinx checks. Let us know if you'd like assistance with moving the commits into the private repo.
Note that the new articles must be added to a TOC.

*IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team
#assign-reviewer: @robinharwood
#assign-reviewer: @Xelu86
#assign: @robinharwood
#assign: @Xelu86

**Comment by robinharwood** [comment]
Hi @gswashington, just to let you know I'm review this today/tomorrow. Please hold off on any edits until I've posted my feedback.

Thanks,
Robin

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+
+## Prerequisites
+
+- Client machine with internet access to run certutil
+- File server or web server for file storage
+- AD Group policy or MDM solution to deploy configuration settings to your client
+
+## Software update description
+
+The following automatic update mechanisms are available in Windows Server 2012 R2 and Windows 8.1 and newer Windows versions or when the appropriate software update is installed (for older operating systems discussed in this article):
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:27]:
```suggestion
The following automatic update mechanisms are available in Windows Server and the Windows client when the appropriate software update is installed (for older operating systems discussed in this article):
```

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+
+- **Tool to select trusted root certificates** This software update introduces a tool for administrators who manage the set of trusted root certificates in their enterprise environment. Administrators can view and select the set of trusted root certificates, export them to a serialized certificate store, and distribute them by using Group Policy. For more information, see the [New Certutil Options]() section in this document.
+
+- **Independent configurability** The automatic update mechanism for trusted and untrusted certificates are independently configurable. Administrators can use the automatic update mechanism to download only the untrusted CTLs and manage their own list of trusted CTLs. For more information, see the [Registry settings modified]() section in this document.
+
+Automatic update functionality also can be disabled if necessary, although this isn't recommended. See [Event ID 8 — Automatic Root Certificates Update Configuration](https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc734054(v=ws.10)?redirectedfrom=MSDN#turn-off-automatic-root-certificates-update) for more information.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:37]: although **this** isn't recommended. - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?

Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:37]:
```suggestion
Automatic update functionality also can be disabled if necessary, although this isn't recommended. For more information, see [Event ID 8 — Automatic Root Certificates Update Configuration](https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc734054(v=ws.10)?redirectedfrom=MSDN#turn-off-automatic-root-certificates-update) for more information.
```

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+
+By using Windows Server 2012 R2 and Windows 8.1 (or by installing the previously mentioned software updates on supported operating systems), an administrator can:
+
+- Configure Active Directory Domain Services (AD DS) domain member computers to use the automatic update mechanism for trusted and untrusted CTLs, without having access to the Windows Update site. This configuration is described in the [Redirect the Microsoft Automatic Update URL for a disconnected environment]() section of this document.
+
+- Configure AD DS domain member computers to independently opt in for untrusted and trusted CTL automatic updates. This configuration is described in the [Redirect the Microsoft Automatic Update URL for untrusted CTLs only]() section of this document.
+
+- Examine the set of root certificates in the Windows Root Certificate Program. This enables administrators to select a subset of certificates to distribute by using a Group Policy Object (GPO). This configuration is described in the [Use a subset of the trusted CTLs]() section of this document.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:59]: Starting the second sentence with this. This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+
+> - If you plan to use a web server, you should create a new virtual directory for the CTL files. The steps to create a virtual directory by using Internet Information Services (IIS) are nearly the same for all the supported operating systems discussed in this document. For more information, see <A href="https://technet.microsoft.com/library/cc771804.aspx">Create a Virtual Directory (IIS7)</A>.
+
+- Certain system and application folders in Windows have special protection applied to them. For example, the <STRONG>inetpub</STRONG> folder requires special access permissions, which make it difficult to create a shared folder for use with a scheduled task to transfer files. As an administrator, you're typically able to create a folder location at the root of a logical drive system to use for file transfers. </LI></UL>
+
+## Redirect the Microsoft Automatic Update URL for a disconnected environment
+
+If the computers in your network are configured in a domain environment and they're unable to use the automatic update mechanism or download CTLs, you can implement a GPO in AD DS to configure those computers to obtain the CTL updates from an alternate location.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:104]: This is quite a complex sentence. Could it be reworded to simplify?

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+
+1. Select **Enabled**. In the **Options** section, enter the URL to the file server or web server that contains the CTL files. For example, `https://server1/CTL` or `file://\\\\server1\\CTL`. select **OK**. Close the Group Policy Management Editor.
+
+The policy is effective immediately, but the client computers must be restarted to receive the new settings, or you can type **gpupdate /force** from an elevated command prompt or from Windows PowerShell.
+
+> [!IMPORTANT]
+> The trusted and untrusted CTLs can be updated on a daily basis, so ensure that you keep the files synchronized by using a scheduled task or another method (such as a script that handles error conditions) to update the shared folder or web virtual directory. For additional details about creating a scheduled task, see [Schedule a Task](https://technet.microsoft.com/library/cc748993.aspx). If you plan to write a script to make daily updates, see the [New Certutil Options]() and [Potential errors with Certutil -SyncWithWU]() sections of this document. These sections provide more information about command options and the error conditions.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:165]:
```suggestion
> The trusted and untrusted CTLs can be updated on a daily basis, so ensure that you keep the files synchronized by using a scheduled task or another method (such as a script that handles error conditions) to update the shared folder or web virtual directory. For more information about creating a scheduled task, see [Schedule a Task](https://technet.microsoft.com/library/cc748993.aspx). If you plan to write a script to make daily updates, see the [New Certutil Options]() and [Potential errors with Certutil -SyncWithWU]() sections of this document. These sections provide more information about command options and the error conditions.
```

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 6]
```diff
@@ -0,0 +1,401 @@
...
+
+## Redirect the Microsoft Automatic Update URL for untrusted CTLs only
+
+Some organizations may want only the untrusted CTLs (not the trusted CTLs) to be automatically updated. To accomplish this, you can create two .adm templates to add to Group Policy.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:169]: `To accomplish this` - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 7]
```diff
@@ -0,0 +1,401 @@
...
+
+| Registry keys | Value and Description |
+|--------|---------|
+| HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\SystemCertificates\AuthRoot\DisableRootAutoUpdate | A value of 1 disables the Windows AutoUpdate of the trusted CTL. |  
+| HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\SystemCertificates\AuthRoot\EnableDisallowedCertAutoUpdate | A value of 1 enables the Windows AutoUpdate of the untrusted CTL. |
+| HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SystemCertificates\AuthRoot\AutoUpdate\RootDirUrl | Configures the shared location (the HTTP or the FILE path). |
+
+## Deleting Trusted and Untrusted CTLs
+
+It may be necessary for various reasons to delete all Trusted and Untrusted CTLs from a client machine. The following Certutil options can be used to accomplish this.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:326]: `can be used to accomplish this.` - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?

File: WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md [hunk 8]
```diff
@@ -0,0 +1,401 @@
...
+
+- If your server is unable to reach the Microsoft Automatic Update servers with the DNS name ctldl.windowsupdate.com, you'll receive the following error:
+
+    The server name or address couldn't be resolved 0x80072ee7 (INet: 12007 ERROR\_INTERNET\_NAME\_NOT\_RESOLVED).
+
+- If you don't use the **-f** switch, and any of the CTL files already exist in the directory, you'll receive a *file* exists error:
+
+    CertUtil: -syncWithWU command FAILED: 0x800700b7 (WIN32/HTTP: 183 ERROR\_ALREADY\_EXISTS) Certutil: Can't create a file when that file already exists.
+
+- If there's a change in the trusted root certificates, you'll see: "Warning\! Encountered the following no longer trusted roots: \<folder path\>\\\<thumbprint\>.crt. Use "-f -f" options to force the delete of the above ".crt" files. Was "authrootstl.cab" updated? If yes, consider deferring the delete until all clients have been updated."
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md:375]: `Use "-f -f" options to force the delete of the above ".crt" files.` Can we replace or provide more locational information to the word 'above'. I'm not even sure the word above needs to be there?

File: WindowsServerDocs/identity/ad-cs/certificates-and-trust.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+Trusted and untrusted root certificates are used by Windows operating systems and applications as a reference when determining whether public key infrastructure (PKI) hierarchies and digital certificates are trustworthy. Untrusted root certificates are certificates that are publicly known to be fraudulent. Trusted and untrusted root certificates functionality works across all environments, whether connected or disconnected.
+
+Trusted and untrusted root certificates are contained in a certificate trust list (CTL). When you want to distribute root certificates you use a CTL. Windows Server features automatic daily update functionality that includes downloads of latest CTLs. The list of trusted and untrusted root certificates are called the Trusted CTL and Untrusted CTL, respectively. See [Announcing the automated updater of untrustworthy certificates and keys](https://blogs.technet.com/b/pki/archive/2012/06/12/announcing-the-automated-updater-of-untrustworthy-certificates-and-keys.aspx) for more information.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/certificates-and-trust.md:20]:
```suggestion
Trusted and untrusted root certificates are contained in a certificate trust list (CTL). When you want to distribute root certificates, you use a CTL. Windows Server features automatic daily update functionality that includes downloads of latest CTLs. The list of trusted and untrusted root certificates are called the Trusted CTL and Untrusted CTL, respectively. See [Announcing the automated updater of untrustworthy certificates and keys](https://blogs.technet.com/b/pki/archive/2012/06/12/announcing-the-automated-updater-of-untrustworthy-certificates-and-keys.aspx) for more information.
```

File: WindowsServerDocs/identity/ad-cs/certificates-and-trust.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+Servers and clients access the Windows Update site to update the CTL using the automatic daily update mechanism (CTL updater) discussed in this article. You can take advantage of CTL updater functionality by installing the appropriate software updates. See [Configure Trusted Roots and Disallowed Certificates](configure-trusted-roots-disallowed-certificates.md) for guidance in installing the software updates on supported operating systems discussed in this article.
+
+## Automatic certificate trust list updates
+
+By default, Windows downloads the CTLs from the Internet via an automatic mechanism called the CTL Updater. The public URLs used by the CTL Updater can be made available to clients. These are:
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/certificates-and-trust.md:26]: ` These are:` - These isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?

File: WindowsServerDocs/identity/ad-cs/certificates-and-trust.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+- http://ctldl.windowsupdate.com/msdownload/update/v3/static/trustedr/en/disallowedcertstl.cab 
+- http://ctldl.windowsupdate.com/msdownload/update/v3/static/trustedr/en/authrootstl.cab
+
+Automatic update functionality also can be disabled if necessary, although this isn't recommended.
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-cs/certificates-and-trust.md:31]:
```suggestion
Automatic update functionality also can be disabled if necessary, although isn't recommended.
```

**Review by robinharwood — CHANGES_REQUESTED** [review]
Just a few minor changes 😄 thanks for your hard work on this @gswashington

**Pull request merged**